src/requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
 pytest==6.2.2
+pytest-profiling==1.7.0
```

Comment:
Is this used anywhere?

Reply:
https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/pull/37/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R30

Reply:
or you can use it locally to measure time of your tests